package.json

```diff
@@ -2,10 +2,10 @@
   "name": "project-html-css",
   "version": "1.1.0",
   "description": "",
-  "homepage": "https://github.com/xertaron/project-html-css",
+  "homepage": "https://github.com/anita992/project-flower",
```

Comment:
Tego nie zmieniasz. To adres repozytorium i jeśli będzie inny niż w pozostałych częściach projektu to skrypt nie będzie wiedział co zrobić.